scalableminds/webknossos
# Upgrade to Play 2.5 and Silhouette 4

This now consists of the changes to our uses of the stream API (originally committed here) as well as the PRs on logging (#3168), Silhouette 4 (#3160), content-disposition (#3171)

Users will be logged out on deploy, as we use new cookies. Rollback should use the old cookies again, roll-forward after that, the installed new ones.

### URL of deployed dev instance (used for testing):
- https://playstreams.webknossos.xyz

### Steps to test:
- all of the backend routes should be tested
- special focus on IO like uploading/downloading zips/nmls
- login should still work
- create a project, trace and finish some tasks, view it, download it
- [standalone datastore setup](https://github.com/scalableminds/webknossos/wiki/Set-up-a-standalone-datastore-locally) should also still work (not block data requests because CORS etc)

### Issues:
- fixes #3103
- fixes #3159
- fixes #3151 
- fixes #3103

------
